### DISCUSSION TITLE
How to use ActivityPub?

### DISCUSSION THREAD

**Discussion by KaabUl** [Automattic/wordpress-activitypub#113] [Q&A]
I installed the ActivityPub plugin on WordPress. But I don't know how to use it.

**Comment by pfefferle** [comment]
Sorry for that 😞  (I plan to invest in usability for the next versions!)

You can find your fediverse identifier/handle in your profile/author settings (username@domain.tld). Others can now follow you on mastodon & co. using that handle to see your blogposts.

**Reply by pfefferle** [reply]
I also work on a better way to help users to find out about possible installation/configuration issues. It is a very complex topic and there is a lot that could go wrong on installtions in shared hosting environments.

**Reply by craigconstantine** [reply]
Could the plugin use PHP curl to perform a webfinger request on an admin user?

**Comment by Cambridgeport90** [comment]
it's fairly simple... I have had it on several installations without issue.
